x-pack/plugins/enterprise_search/public/application.test.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This should be renamed `index.test.ts`

Reply:
Ah, and should also be moved to `public/applications/index.test.ts`

Reply:
a7b0acd